README.md

```diff
@@ -75,6 +77,35 @@ generic with minimal need for project-specific behavior.
 [Docker]: https://docs.docker.com/install/overview/
 
 
+### 2.1.1 git-secrets
+
+[git-secrets] helps to prevent secrets (passwords, credentials, etc.) from being
+submitted into a git repository. See the "installing git-secrets" section of the
+project's README.rst for instructions how to install git-secrets on your OS.
+
+Once installed, git-secrets will need to be configured for your local Azul
+repositories. To configure git-secrets in a single local repository, run:
+
+    cd /path/to/my/repo
+    git secrets --install
+    git secrets --register-aws
+    
+Alternatively, you can configure git-secrets globally for all existing
+repositories and those you will initialize or clone in the future:
```

Comment:
I don't think this installs it globally for all existing repos. It just does it for clones created in the future.